apps/portal/src/modules/collections/actions/helpers.ts

```
@@ -281,13 +287,13 @@ export async function upsertSeoMeta({
 /**
- * Soft deletes SEO metadata
- * @param seoId - The SEO ID to delete
- * @param transaction - Database transaction (optional)
+ * Marks the specified SEO metadata as deleted by setting its deletedAt and updatedAt timestamps.
+ *
+ * @param seoId - The ID of the SEO metadata to soft delete
  */
 export async function softDeleteSeo(
   seoId: string,
-  transaction: any = db,
+  transaction: any = db
```

Biome (2.1.2)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)